woodwork/tests/schema/test_table_schema.py

```diff
@@ -555,9 +555,9 @@ def test_set_logical_types(sample_column_names, sample_inferred_logical_types):
     assert schema.logical_types["age"] == Double
 
     # Verify semantic tags were reset to standard tags in columns with Logical Type changes
-    assert schema.semantic_tags["full_name"] == {"category"}
-    assert schema.semantic_tags["email"] == set()
-    assert schema.semantic_tags["phone_number"] == set()
+    assert schema.semantic_tags["full_name"] == {"category", "tag1"}
```

Comment:
update this comment to make it clear that we dont want the custom tags to be reset